### ISSUE TITLE
REGRESSION: Bad node binary in 7.167.1 (Sept 1, 2022 release candidate)

### ISSUE THREAD

**Issue by daveespo** [forcedotcom/cli#1696]
### Summary

Our nightly CI build tries to use the latest Release Candidate to build a scratch org. It downloads the tarball installer from:

https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-linux-x64.tar.xz

Upon unpacking the tarball, the `bin/sfdx` script exits as follows:

```
esposito@PT-WIN-ESPOSITO:~$ sfdx/bin/sfdx
sfdx/bin/sfdx: line 44: /home/esposito/sfdx/bin/node: cannot execute binary file: Exec format error
```

### Steps To Reproduce:

Download the above tarball to an Ubuntu environment

Unpack the tarball

Execute `sfdx/bin/sfdx`

Expected: Command line help

Actual: Exec format error

I confirmed that it's actually the `node` executable in the `bin/` directory that's corrupted

```
esposito@PT-WIN-ESPOSITO:~/sfdx$ bin/node
-bash: bin/node: cannot execute binary file: Exec format error
```

I'm unable to export the version info since the sfdx command won't run ;-)

**Comment by 0ptaq0** [comment]
I am having the same issue during the docker build
```
Step 17/17 : RUN sfdx version
 ---> Running in f4275...bcfe
[91m/usr/local/sfdx/bin/sfdx: line 44: /usr/local/sfdx/bin/node: cannot execute binary file: Exec format error
[0mService 'slave-salesforce' failed to build: The command '/bin/sh -c sfdx version' returned a non-zero code: 126
```

**Comment by pgajek2** [comment]
Yep, the same here. It blocks our development.

**Comment by mshanemc** [comment]
We saw the same and patched it as 7.167.2.  Any better?

**Comment by daveespo** [comment]
Yes, thanks. The past couple of nights' builds have passed.
